nixos/doc/manual/release-notes/rl-2103.xml

```diff
@@ -26,6 +26,22 @@
    <listitem>
     <para>GNOME desktop environment was upgraded to 3.38, see its <link xlink:href="https://help.gnome.org/misc/release-notes/3.38/">release notes</link>.</para>
    </listitem>
+   <listitem>
```

Comment:
In addition to the nixos release notes we should probably mention it in the stdenv docs so new contributors will be able to read up on it. We already document `passthur.updateScript` there so this wouldn't be too weird.

Reply:
Is there a place in the documentation that guides people through how to add a new editor/pager/shell/specific type of package, already? I don't remember seeing anything of that sort, and it'd be more easily discoverable there.

Reply:
I don't think there is how to package XYZ in the documentation but rather documentation on our build system where we already document other attributes of the passthru's. Due to a lack of a better section I'd stick it in there so it isn't undocumented. If, at some point, we have more specific documentation that could be moved.